macros/multiple_databases/charindex.sql

```diff
@@ -0,0 +1,21 @@
+{%- macro charindex(expression_to_find, field, start_location=None) -%}
+case
+    when len('{{ expression_to_find }}') > 0
+    then
+        {% if start_location is none -%}
+            {% if target.type == 'databricks' -%}
+                position('{{ expression_to_find }}', {{ field }})
+            {% else -%}
+                charindex('{{ expression_to_find }}', {{ field }})
+            {% endif -%}
+        {% else -%}
+            {% if target.type == 'databricks' -%}
+                position('{{ expression_to_find }}', {{ field }}, '{{ start_location }}')
```

Comment:
Also the start location is expected to be an integer, in other words it does not work correctly. It runs, it compiles, but the result is incorrect.